gtfu/src/main/java/gtfu/GPSData.java

```diff
@@ -4,11 +4,13 @@
     public long millis;
     public float lat;
     public float lon;
+    public int count;
 
-    public GPSData(long millis, float lat, float lon) {
+    public GPSData(long millis, float lat, float lon, int count) {
```

Comment:
It would be cleaner to set it to 1 and not require count as a parameter.

Reply:
Don't know if I'm understanding correctly, but you can overload constructors:
```
public GPSData(long millis, float lat, float lon) {
    this(long millis, float lat, float lon, 1);
}

public GPSData(long millis, float lat, float lon, int count) {
    ...
}
```